.travis.yml

```diff
@@ -19,11 +19,13 @@ install:
   - tar -xvf $PWD/travis-phantomjs/phantomjs-2.0.0-ubuntu-12.04.tar.bz2 -C $PWD/travis-phantomjs
   - export PATH=$PWD/travis-phantomjs:$PATH
 
+  - cp ./api/.travis.env ./api/.env
+
   - travis_retry npm install --no-spin --quiet -g gulp http-server
   - travis_retry npm install --no-spin
   - travis_retry gulp bower:install
   - travis_retry composer self-update
-  - travis_retry composer install --prefer-source --no-interaction --working-dir api
+  - travis_retry composer install --prefer-source --no-interaction --optimize-autoloader --working-dir api
```

Comment:
@xiphiaz It actually do pickup the namespacing without optimized, but the problem you see come from two classes not being named according to the PSR standard. The SpiraApplication and HandleExceptionsFix classes have filenames that do not comform to PSR. So autoloading does not find them. Renaming them accordingly would allow autoload to work without the optimize flag.

When dumping with optimize, composer parses each file and matches namespace path with filename, and therefore it works despite the difference in naming vs class names.